package.json

```diff
@@ -15,6 +15,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@capacitor/core": "^3.0.1",
+    "ionic": "^5.4.16",
```

Comment:
do we need this dependency? seems like it's causing the massive diff in `package-lock.json`